### PR TITLE
EDU-64 Make dependencies more flexible

### PR DESCRIPTION
**Pull request by ahelmberger**

### CODE REVIEW

File: src/components/pages/edit-doc.js [hunk 1]
```diff
@@ -54,13 +54,9 @@ class EditDoc extends React.Component {
 
     autoBind(this);
 
-    const { editorFactory, rendererFactory, documentApiClient, initialState } = this.props;
+    const { initialState } = this.props;
     const { documentRevision, proposedSections } = initialState;
 
-    this.editorFactory = editorFactory;
```
Comment (laurentiu-ilici) [inline src/components/pages/edit-doc.js:60]: Why are these changes here? It doesn't seem to be part of the story.

Comment (adrianaluchian) [inline src/components/pages/edit-doc.js:60]: Uh, more leftovers from menus 😬

**Review by laurentiu-ilici — APPROVED** [review]

**Review by adrianaluchian — APPROVED** [review]

File: src/components/pages/edit-doc.js (hunk 1 above)
Comment (ahelmberger) [reply src/components/pages/edit-doc.js:60]: Linting rules have changed and it now realizes that this line is not used

Comment (ahelmberger) [reply src/components/pages/edit-doc.js:60]: Now the good linter tells us more about unused code
